Makefile

```diff
@@ -34,19 +36,23 @@ permission:
 
 # -- Downloading and extracting IBGE files
 
-# Downloads the zip files
-# ftp://geoftp.ibge.gov.br/malhas_digitais/municipio_2010/
+# Downloads States zip files
+# ftp://geoftp.ibge.gov.br/organizacao_do_territorio/malhas_territoriais/malhas_municipais/municipio_2021/UFs/
 zip/%.zip:
 	$(eval STATE := $(patsubst %-municipalities,%,$*))
 	$(eval STATE := $(patsubst %-micro,%,$(STATE)))
 	$(eval STATE := $(patsubst %-meso,%,$(STATE)))
+	$(eval STATE := $(patsubst %-immediate,%,$(STATE)))
+	$(eval STATE := $(patsubst %-intermediate,%,$(STATE)))
 	$(eval STATE := $(patsubst %-state,%,$(STATE)))
-	$(eval FILENAME := $(subst -municipalities,_municipios,$*))
-	$(eval FILENAME := $(subst -micro,_microrregioes,$(FILENAME)))
-	$(eval FILENAME := $(subst -meso,_mesorregioes,$(FILENAME)))
-	$(eval FILENAME := $(subst -state,_unidades_da_federacao,$(FILENAME)))
+	$(eval FILENAME := $(subst -municipalities,_Municipios_2020,$*))
```

Comment:
Você atualizou o link para os shapes `2021`mas aqui está indicado `2020`. Acho que vale extrair o "ano" para uma variável e usar tanto aqui quanto na URL.

Reply:
Indeed, thanks for the heads up. As soon as I can I look into it and do the changes for now I will fix this part.